sonar-scanner-engine/src/main/java/org/sonar/scanner/scan/ProjectScanContainer.java

```diff
@@ -239,7 +239,8 @@ protected void doAfterStart() {
 
     String branchName = props.property(ScannerProperties.BRANCH_NAME);
     if (branchName != null) {
-      LOG.info("Branch name: {}", branchName);
+      BranchConfiguration branchConfig = getComponentByType(BranchConfiguration.class);
+      LOG.info("Branch name: {}, type: {}", branchName, branchConfig.branchType().toString().toLowerCase());
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Define the locale to be used in this String operation. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1449)